jarviz-lib/pom.xml

```diff
@@ -29,7 +29,7 @@
 
     <properties>
```

Comment:
@hadisfr - I just finished migrating to GH Action and as part of it I upgraded libs and fixed some warnings. So I just realized you also did the same here. We have latest ASM version 9.1 now with the release of 0.1.4.

Reply:
So this is already done. :D